src/order/category/FinPartialOrder.lean

```diff
@@ -0,0 +1,76 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import category_theory.Fintype
+import order.category.PartialOrder
+
+/-!
+# The category of finite partial orders
+
+This defines `FinPartialOrder`, the category of finite partial orders.
+
+Note: `FinPartialOrder` is NOT a subcategory of `BoundedOrder` because its morphisms do not
+preserve `⊥` and `⊤`.
```

Comment:
More to the point, finite partial orders don't need to be bounded.

Reply:
Oh whoops. I had the case of finite lattices in mind.